walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -55,7 +55,7 @@
           Set<String> optAttendees, long duration) {
     ArrayList<TimeRange> busyTimes = new ArrayList<TimeRange>();
     ArrayList<TimeRange> optionalBusyTimes = new ArrayList<TimeRange>();
-    for(Event event : events) {
+    /*for(Event event : events) {
```

Comment:
why are you keeping in this commented code?

Reply:
I'm not-- if you look at the most recent commits I removed this & it should all be much cleaner overall